executor/envd/server.py

```
@@ -0,0 +1,426 @@
+        )
+    except Exception as e:
+        logger.exception(f"Error handling unary RPC: {e}")
+        return Response(
+            content=json.dumps({"code": "internal", "message": str(e)}),
+            status_code=500,
+            media_type="application/json",
+            headers={"Connect-Protocol-Version": "1"}
+        )
+
+
+async def _handle_server_stream(request: Request, handler, req_class, resp_class):
```

Ruff (0.14.11)

Unused function argument: `resp_class`

(ARG001)